src/bundle-stats.js

```diff
@@ -153,6 +158,42 @@ BundleStatsCollector.prototype.AddDebugFile = function (bundleName, fileName) {
     this.DebugCollection.add(bundleName, fileName);
 };
 
+BundleStatsCollector.prototype.AddExport = function (bundleName, exportName, filePath) {
+
+    var exp = {};
+    exp[exportName] = filePath;
+    this.ExportsCollection.add(bundleName, exp);
+
+};
+
+BundleStatsCollector.prototype.GetExportsForBundle = function(bundleName) {
+
+    return this.ExportsCollection.get(bundleName);
+
+};
+
+BundleStatsCollector.prototype.ParseJsonForStats = function (bundleName, filePath, text) {

```

Comment:
When we encounter a `package.json` file, this parses it to determine what modules to expose globally on `require`. This gets used down the chain in `pack-files` as part of `requireify`.